main/gen_gdl90.go

```diff
@@ -1187,6 +1187,7 @@ type settings struct {
 	SensorQuaternion     [4]float64 // Quaternion mapping from sensor frame to aircraft frame
 	C, D                 [3]float64 // IMU Accel, Gyro zero bias
 	PPM                  int
+	Gain                 float64
```

Comment:
Mind giving this a bit more descriptive name? There are a lot of gain values in Stratux.. a reader would not know which one this controls. Maybe Dump1090Gain or something.